File: fieldsync-inventory-backend/src/main/java/com/example/fieldsync_inventory_backend/dto/sales/customer_type/CustomerTypeResponseDTO.java
package com.example.fieldsync_inventory_backend.dto.sales.customer_type;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.Instant;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class CustomerTypeResponseDTO {
    private Integer id;
    private String name;
    private String description;
    private Instant createdAt;
    private Instant updatedAt;
    private Instant deletedAt;
}

File: fieldsync-inventory-app/app/src/main/java/com/example/fieldsync_inventory_app/data/remote/dto/customer_type/CustomerTypeResponseDto.kt
package com.example.fieldsync_inventory_app.data.remote.dto.customer_type

data class CustomerTypeResponseDto(
    val id: Int,
    val name: String,
    val description: String?,
    val createdAt: String,
    val updatedAt: String,
    val deletedAt: String?
)